package.json

```diff
@@ -39,13 +51,15 @@
     "apidocs": "jsdoc -c jsdoc.json",
     "cppcheck": "cppcheck --quiet --enable=warning,style -I src/include src/main/",
     "valgrind": "valgrind node ./node_modules/mocha/bin/_mocha -R dot -g '#slow' -i",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "__comment__": "prepare: husky install"
   },
   "dependencies": {
     "bindings": "^1.3.0",
     "minimatch": "^3.0.4",
     "nan": "^2.14.0",
-    "node-gyp": "^8.4.1"
+    "node-gyp": "^8.4.1",
+    "node-pre-gyp": "^0.17.0"
```

Comment:
Should use `@mapbox/node-pre-gyp` instead:

> On Feb 9th, 2021 @mapbox/node-pre-gyp@1.0.0 was [released](https://github.com/mapbox/node-pre-gyp/blob/HEAD/CHANGELOG.md). Older, unscoped versions that are not part of the @mapbox org are deprecated and only @mapbox/node-pre-gyp will see updates going forward.

https://www.npmjs.com/package/@mapbox/node-pre-gyp